utils/release/reify.mjs

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env node
+import Arborist from '@npmcli/arborist';
+import {DepGraph} from 'dependency-graph';
+import {dirname, resolve} from 'node:path';
+import {fileURLToPath} from 'node:url';
+
+/**
+ * Current strategy: reify each package (along with their dependants transitively) in topological (parents->dependants) order.
+ *
+ * Other (untested) strategies that may work:
+ * * Strategy A: reify each package individually in topological (parents->dependants) order.
+ * * Strategy B: reify all packages at once.
+ */
```

Comment:
This script is probably overly complex, since strategy A or strategy B may also work.

However, I've wasted enough time on this, this works very well and we may eventually switch to `pnpm` anyway, so YOLO.